src/test/java/org/example/action/user/FullFormAction.java

```diff
@@ -33,10 +33,13 @@
 @Action
 public class FullFormAction {
   public List<Integer> ages = new ArrayList<Integer>();
+  private static List<Integer> agesFromLastInvocation;
```

Comment:
Can you run the formatter on this file?

Reply:
Sorry. I thought I had it on for this project, swore I did but it's off.